src/ansys/mapdl/core/mapdl_geometry.py

```diff
@@ -192,28 +595,18 @@ def areas(self, quality=1, merge=False):
           N Arrays:	4,
         ...
 
-        Return a single merged mesh.
-
-        >>> area_mesh = mapdl.areas(quality=3)
-        >>> area_mesh
-        UnstructuredGrid (0x7f14add95ca0)
-          N Cells:	24
-          N Points:	30
-          X Bounds:	-2.000e+00, 2.000e+00
-          Y Bounds:	0.000e+00, 1.974e+00
-          Z Bounds:	5.500e-01, 5.500e-01
-          N Arrays:	4
-
-
         """
         quality = int(quality)
         if quality > 10:
             raise ValueError("``quality`` parameter must be a value between 0 and 10")
```

Comment:
No concluding period. Perhaps start this string with "The ``quality`` parameter.... so as not to start with a literal?